cmd/services.rb

```diff
@@ -41,14 +41,17 @@ def services_args
   def services
     args = services_args.parse
 
-    raise UsageError, "`brew services` is supported only on macOS!" unless OS.mac?
-
     # pbpaste's exit status is a proxy for detecting the use of reattach-to-user-namespace
     raise UsageError, "`brew services` cannot run under tmux!" if ENV["TMUX"] && !quiet_system("/usr/bin/pbpaste")
```

Comment:
```suggestion
    raise UsageError, "`brew services` cannot run under tmux!" if ENV["HOMEBREW_TMUX"] && !quiet_system("/usr/bin/pbpaste")
```
May as well fix this while we're here.